docs/usage/10.md

```diff
@@ -14,9 +14,70 @@ The phases where hooks may be run are:
 # Default hook image
 The default hook image is `quay.io/konveyor/hook-runner:latest`. It is based off of the Ansible Runner image with the addition of python-openshift to provided Ansible k8s resources as well as a recent oc binary.
 
+# Writing ansible for hooks
+When writing ansible to run within the pod it is possible to use the `oc` command or the k8s modules to accomplish tasks. A token is mounted in the container for the specified service account so there is no need to manually authenticate before running tasks in the cluster.
+
+```
+- hosts: localhost
+  gather_facts: false
+  tasks:
+  - name: get pod name
+    shell: oc get po --all-namespaces
+```
+
+```
+- hosts: localhost
+  gather_facts: false
+  tasks:
+  - name: Get pod
+    k8s_facts:
+      kind: pods
+      api: v1
+      namespace: openshift-migration
+      name: "{{ lookup( 'env', 'HOSTNAME') }}"
+    register: pods
+
+  - name: Print pod name
+    debug:
+      msg: "{{ pods.resources[0].metadata.name }}"
+```
+
+# Special Environment variables
+The migplan name and associated namespaces are passed in as environment variables to the job container so they can be used in writing tasks. To access use an env lookup.
+
+For example:
+```
+- hosts: localhost
+  gather_facts: false
+  tasks:
+  - debug:
+      msg: "{{ (lookup( 'env', 'MIGRATION_NAMESPACES')).split(',') }}"
```

Comment:
Do you think it would be useful if we provided an example of looping over these namespaces and taking an action on each of them?

Reply:
Sure I set a fact and used with_items to print the them each instead of the whole list in one go. I think that would be a general pattern someone would use if trying to do something with each.